### PR TITLE
Fix CI status badge on README

### PR DESCRIPTION
**Pull request by tuliohmendes**
## 📑 What does this PR do?

Small fix on the `README.md` file to show the correct CI status badge from GitHub Actions, as per documentation: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

# ✅ Checklist

- [X] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [X] All the tests have passed

## 🧪 How can this PR be tested?

Verify that the rendered README on GitHub shows the proper status.

## 🖼️ Screenshots (if applicable):

image

### CODE REVIEW

**Review by Rafsanjani — APPROVED** [review]

File: README.md
Comment (crocsandcoffee) [inline README.md:3]: This will break again once the following PR gets merged, which changes the name of the script
https://github.com/AndroidDev-social/MastodonCompose/pull/41

**Review by crocsandcoffee — CHANGES_REQUESTED** [review]
Please wait for https://github.com/AndroidDev-social/MastodonCompose/pull/41 to get merged and update the changes in this PR

**Comment by tuliohmendes** [comment]
> Please wait for #41 to get merged and update the changes in this PR

Hey @crocsandcoffee 👋

Makes sense. I will wait for the PR to get merged.

File: README.md
Comment (thebino) [reply README.md:3]: Since it is not merged yet and will not soon without changes, we can approve this one.
Taking small steps is always better, then none

**Review by thebino — APPROVED** [review]

File: README.md
Comment (crocsandcoffee) [reply README.md:3]: Okay, then let's just make sure this is updated on that other PR that is open or done as a follow-up right away. I prefer to avoid churn on small stuff like this and just get it fixed and over with.

Comment (thebino) [reply README.md:3]: approve and merge it, I'll have a look into the scripts

**Review by crocsandcoffee — APPROVED** [review]

**Pull request merged**
